article/app/controllers/ArticleController.scala

```diff
@@ -194,6 +196,12 @@ class ArticleController(contentApiClient: ContentApiClient, val controllerCompon
 
   }
 
+  def timedPage[T](future: Future[T], metric: TimingMetric): Future[T] = {
```

Comment:
Could give this an even more generic name given it doesn't really care what it is timing?